src/domain/validations.ts

```diff
@@ -147,7 +180,7 @@ export function validatePrimaryAffiliationMatching(
   }
 
   errors.push({
-    field: 'primaryAffililation',
```

Comment:
Typo fix. I didn't see any other references for this besides the state test file, do you know if this is referenced anywhere else?